shivyc/il_cmds/base.py

```diff
@@ -139,8 +139,14 @@ def _is_imm(self, spot):
         """Return True iff given spot is an immediate operand."""
         return isinstance(spot, LiteralSpot)
 
+    def _is_imm8(self, spot):
+        """Return True if given spot is a 8-bit immediate operand."""
+        return (self._is_imm(spot) and
+                (int(spot.detail) > ctypes.unsig_char_max or
```

Comment:
I don't think this is what you meant - the second condition just checks whether `int(spot.detail)` is not equal to `ctypes.unsig_char_max`